### DISCUSSION TITLE
Chiusura Enel X Homix - Alternative gestione valvole wireless e caldaia

### DISCUSSION THREAD

**Discussion by Foofighterclub** [indomus/forum#3185] [GENERALE - Discussioni generiche]
Adesso che il servizio di EnelX, Homix, verrà chiuso ad Ottobre, per chi aveva abbinate con Homix le valvole termostatiche Danfoss Ally, che alternativa c'è adesso? Gateway e relè caldaia Danfoss?
L'Amazon Echo Hub, avendo la tecnologia Zigbee, potrebbe essere compatibile con le valvole?
Grazie e buona giornata.

**Comment by Scialla** [comment]
Se hai già il gateway, su alexa c'è la skill Danfoss e sul sito dice che sono compatibili con Alexa..

**Reply by Foofighterclub** [reply]
Gateway Danfoss, intendi?

**Reply by Foofighterclub** [reply]
Sto usando altre skill Alexa di altri prodotti, ma non mi sembrano tanto funzionanti.

**Reply by Scialla** [reply]
si il Danfoss, presumo che, tramite quello, la skill di Alexa veda le teste termostatiche e le riesca a pilotare
